cmd/solvercli/cmd/settle_orders.go

```diff
@@ -0,0 +1,153 @@
+	if err != nil {
+		lmt.Logger(ctx).Error("Failed to get sqlite-db-path", zap.Error(err))
+		return
+	}
+
+	migrationsPath, err := cmd.Flags().GetString("migrations-path")
+	if err != nil {
+		lmt.Logger(ctx).Error("Failed to get migrations-path", zap.Error(err))
+		return
+	}
+
+	configPath, err := cmd.Flags().GetString("config")
+	if err != nil {
+		lmt.Logger(ctx).Error("Failed to get config path", zap.Error(err))
+		return
+	}
+
+	cfg, err := config.LoadConfig(configPath)
+	if err != nil {
+		lmt.Logger(ctx).Error("Unable to load config", zap.Error(err))
+		return
+	}
+
+	ctx = config.ConfigReaderContext(ctx, config.NewConfigReader(cfg))
+
+	keyStore, err := keys.GetKeyStore(keyStoreType, keys.GetKeyStoreOpts{KeyFilePath: keysPath})
+	if err != nil {
+		lmt.Logger(ctx).Fatal("Unable to load keystore", zap.Error(err))
+	}
+
+	cosmosTxExecutor := cosmos.DefaultSerializedCosmosTxExecutor()
+	evmTxExecutor := evm.DefaultEVMTxExecutor()
+
+	clientManager := clientmanager.NewClientManager(keyStore, cosmosTxExecutor)
+
+	dbConn, err := connect.ConnectAndMigrate(ctx, sqliteDBPath, migrationsPath)
+	if err != nil {
+		lmt.Logger(ctx).Fatal("Unable to connect to db", zap.Error(err))
+	}
+	defer dbConn.Close()
+
+	evmManager := evmrpc.NewEVMRPCClientManager()
+	rateLimitedClient := utils.DefaultRateLimitedHTTPClient(3)
+	coingeckoClient := coingecko.NewCoingeckoClient(rateLimitedClient, "https://api.coingecko.com/api/v3/", "")
+	cachedCoinGeckoClient := coingecko.NewCachedPriceClient(coingeckoClient, 15*time.Minute)
+	evmTxPriceOracle := evmrpc.NewOracle(cachedCoinGeckoClient)
+
+	hype, err := hyperlane.NewMultiClientFromConfig(ctx, evmManager, keyStore, evmTxPriceOracle, evmTxExecutor)
+	if err != nil {
+		lmt.Logger(ctx).Fatal("creating hyperlane multi client from config", zap.Error(err))
+	}
+
+	relayer := hyperlane.NewRelayer(hype, make(map[string]string))
+	relayerRunner := hyperlane.NewRelayerRunner(db.New(dbConn), hype, relayer)
+
+	settler, err := ordersettler.NewOrderSettler(ctx, db.New(dbConn), clientManager, relayerRunner)
+	if err != nil {
+		lmt.Logger(ctx).Error("creating order settler", zap.Error(err))
+		return
+	}
+
+	batches, err := settler.PendingSettlementBatches(ctx)
+	if err != nil {
+		lmt.Logger(ctx).Error("getting pending settlement batches", zap.Error(err))
+		return
+	}
+
+	if len(batches) == 0 {
+		fmt.Println("No pending settlement batches found")
+		return
+	}
+
+	fmt.Printf("Found %d pending settlement batches\n", len(batches))
+
+	hashes, err := settler.SettleBatches(ctx, batches)
+	if err != nil {
+		lmt.Logger(ctx).Error("settling pending batches", zap.Error(err))
+		return
+	}
+
+	for i, batch := range batches {
+		fmt.Printf("Settled batch %d:\n", i+1)
+		fmt.Printf("Source Chain: %s\n", batch.SourceChainID())
+		fmt.Printf("Destination Chain: %s\n", batch.DestinationChainID())
+		fmt.Printf("Number of Orders: %d\n", len(batch.OrderIDs()))
+		fmt.Printf("Transaction Hash: %s\n", hashes[i])
```

Comment:
Here too

Reply:
I personally think the format is clearer when the final result is printed to the user this way rather than a structured log, that's why I used fmt.Printf for these 3 instances, but if you disagree I can change too